app/components/properties-panel/properties-panel.component.html

```diff
@@ -55,7 +55,13 @@
 
                             <template [ngIf]="pair[1].subtype(Type.Node)">
                                 <label  class="node-label">
-                                    {{selectedEntity[group[1]].get(pair[0]).drawableProperties.get('Label')}}
+                                    <!-- TODO: Figure out a better way to allow nodes to sometimes be null
+                                        
+                                        Specifically, this doesn't have a value yet, and this breaks a lot of 
+                                        promises that say that there is always a start state, that is, we can't 
```

Comment:
Wait I don't think we should say there is always a start state. Or am I misunderstanding you? When you say optional do you mean an Optional<of> type or a more colloquial optional? Because I think start state should definitely be an Optional<Type.Node> property. (If it's too hard for that plumbing we can just assume all properties can be null and draw them accordingly in the property's panel)
It's fine to special case stuff here now, because I'm going to make this all generic soon.